app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CategoriesController < ApplicationController
+
+  def index
+    @categories = Category.all
+ end
```

Comment:
Indent lines, pls. NOTE: on real projects we might run a Ruby linter to ensure that this is done automatically. However, I highlight it so that you become aware that it's a real thing:tm: that people are about.